npm/wizer/README.md

```diff
@@ -0,0 +1,18 @@
+# wizer
+
+> Prebuilt wizer binaries available via npm
+
+## API
+
+```
+$ npm install --save @bytecode-alliance/wizer
+```
+
+```js
+const execFile = require('child_process').execFile;
+const wizer = require('@bytecode-alliance/wizer');
+
+execFile(wizer, ['input.wasm', '-o', 'initialized.wasm'], (err, stdout) => {
+	console.log(stdout);
+});
```

Comment:
Perhaps make it an API? Simple version could just wrap the temp file process?

```js
import wizer from '@bytecode-alliance/wizer';

const inputBytes = fs.readFileSync('input.wasm');
const outputBytes = await wizer(inputBytes);
fs.writeFileSync('output.wasm', outputBytes);
```

Another benefit would be that instead of returning the path, a runtime lookup could be done avoiding a postinstall.